### PR TITLE
Normalization crazyness

### PR DESCRIPTION
**Pull request by leogrim**
@yingjieMiao @eishay

### CODE REVIEW

**Comment by leogrim** [comment]
@raymondng found two major problems here, the one fixed in Scraper is probably going to make our life a lot better ;)

**Comment by yingjieMiao** [comment]
lgtm

File: kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeWorker.scala [hunk 1]
```diff
@@ -118,7 +118,8 @@ class ScrapeWorker(
     val Scraped(article, signature, redirects) = scraped
     val updatedUri = processRedirects(latestUri, redirects)
 
-    if (!needReIndex(latestUri, updatedUri, article, signature, info)) {
+    if (updatedUri.state == NormalizedURIStates.REDIRECTED) (updatedUri, None)
```
Comment (raymondng) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeWorker.scala:121]: :+1: Long overdue

Comment (raymondng) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeWorker.scala:121]: I wonder about this though ... if it's indeed a long standing issue then we should have observed its effects for a while (and not just the last few days), right?

Comment (yingjieMiao) [inline kifi-backend/modules/scraper/app/com/keepit/scraper/ScrapeWorker.scala:121]: @leogrim 

we should do something about scrapeInfo for the redirected uri.

**Pull request merged**